### PR TITLE
chore(TESTING.md): mention integration branches

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

**Comment by jkomyno** [comment]
Thank you for this documentation. I would also add a brief paragraph for Prisma employees only, something among the lines "Check out the #feed-prisma-releases channel on Slack to get notified when the integration version of `prisma` is ready."

**Review by jkomyno — APPROVED** [review]
Great work, please see whether my comment above can be integrated as well

**Comment by jkomyno** [comment]
Great, feel free to merge :)

File: TESTING.md [hunk 1]
```diff
@@ -123,4 +123,23 @@ By creating a Pull Request the following pipelines will be triggered
...
+
+Once published to npm the version will need to be installed with the exact version like:
+
+```
+npm install -D prisma@3.12.0-fix-all-the-things.1
+
+# or executed with npx like
+npx prisma@3.12.0-fix-all-the-things.1
+```
+
```
Comment (janpio) [inline TESTING.md:144]: (Note that Npm version upgrades or the update notifier in Prisma CLI might behave weird and unexpected with these integration versions.)

**Review by janpio — APPROVED** [review]

**Pull request merged**